xml/alerts-2.5.rng

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grammar xmlns="http://relaxng.org/ns/structure/1.0" 
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+  <start>
+      <ref name="element-alerts"/>
+  </start>
+
+  <define name="element-alerts">
+    <optional>
+    <element name="alerts">
+      <zeroOrMore>
+        <element name="alert">
+          <attribute name="id"><text/></attribute>
+          <!-- path to the script called for alert -->
+          <attribute name="path"><text/></attribute>
+          <optional>
+            <attribute name="timeout"><text/></attribute>
+          </optional>
+          <optional>
+            <!-- format string fed into strftime defaults to %s -->
+            <!-- use format as with strftime + %N as with date -->
+            <attribute name="tstamp_format"><text/></attribute>
+          </optional>
+          <zeroOrMore>
+            <element name="environment_variables">
```

Comment:
... and of course all the good and the bad coming from defining things more loosely via
the schema. We don't have anything like the resouce-meta-data being used by e.g. pcs
to verify which instance attributes are being added ...